requirements.txt

```diff
@@ -0,0 +1 @@
+ladybug-geometry==1.6.2
```

Comment:
I think this makes sense to keep the requirements to a released version rather than the development branch but I think @chriswmackey is using the development branch here for a reason.